doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -44,354 +44,1448 @@ This resource allows you to obtain a list of subresources published by the API:
-    + type: Room (required, string) - Entity Type
+    + entityId: 0115206c51f60b48b77e4c937835795c33bb953f (required, string) - Entity ID
+    + type: Restaurant (required, string) - Entity Type
     + options: keyValues (optional, string) - rendering options
 
++ Request 
+    
+    + Headers
+
+            Fiware-Service: tourguide
+
 + Response 200 (application/json)
 
+
         {
-            "type": "Room",
-            "id": "Boe_Idearium",
-            "temperature": 22.5
+          "id": "0115206c51f60b48b77e4c937835795c33bb953f",
+          "type": "Restaurant",
+          "address":
+          {
+            "streetAddress": "Cuesta de las Cabras Aldapa 2",
+            "addressRegion": "Araba",
+            "addressLocality": "Alegría-Dulantzi",
+            "postalCode": "01240"
+          },
+          "aggregateRating":
+          {
+            "reviewCount": 1,
+            "ratingValue": 3
+          },
+          "capacity": 120,
+          "department": "Franchise2",
+          "description": "Restaurante de estilo sidrería ubicado en Alegria-Dulantzi. Además...",
+          "location": "42.8404625, -2.5123277",
+          "name": "Elizalde",
+          "occupancyLevels": 0,
+          "priceRange": 0,
+          "telephone": "945 400 868"
         }
 
+
+
 ## Partial entity retrieval [/v2/entities/{entityId}{?attrs}]
 
 ### One entity attribute [GET /v2/entities/{entityId}{?attrs}]
 
 + Parameters
-    + entityId: Bcn_Welt (required, string) - Entity ID
-    + attrs: temperature (optional, string) - Attribute to be retrieved
+    + entityId: 0115206c51f60b48b77e4c937835795c33bb953f (required, string) - Entity ID
+    + attrs: name (optional, string) - Attribute to be retrieved
+
++ Request 
+    
+    + Headers
+
+            Fiware-Service: tourguide
 
 + Response 200 (application/json)
 
         {
-            "type": "Room",
-            "id": "Boe_Idearium",
-            "temperature": {
-              "value":    22.7,
-              "type":     "urn:ogc:def:phenomenon:temperature",
-              "metadata": { }
+          "id": "0115206c51f60b48b77e4c937835795c33bb953f",
+          "type": "Restaurant",
+          "name":
+          {
+            "type": "none",
+            "value": "Elizalde",
+            "metadata":
+            {
             }
+          }
         }
 
+
+
 ### One entity attribute. Not present [GET /v2/entities/{entityId}{?attrs}]
 
 + Parameters
-    + entityId: Bcn_Welt (required, string) - Entity ID
+    + entityId: 0115206c51f60b48b77e4c937835795c33bb953f (required, string) - Entity ID
     + attrs: mistake (optional, string) - Attribute to be retrieved
 
-+ Response 200 (application/json)
++ Request 
+    
+    + Headers
+
+            Fiware-Service: tourguide
+
++ Response 404 (application/json)
 
         {
-           "type": "Room",
-           "id": "Boe_Idearium"
+          "error": "NotFound",
+          "description": "The requested entity has not been found. Check type and id"
```

Comment:
Is this the real output from Orion? Shouldn't the description state something like 'The requested attribute has not been found.[...]'